coordinator/auth-api/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>stargate</artifactId>
-    <version>2.1.0-BETA-20-SNAPSHOT</version>
+    <version>2.1-CNDB-7343-v2.1-SNAPSHOT</version>
```

Comment:
These need to be rolled back before merging.

Reply:
Yes, they were to accommodate CI testing.